tests/unit/helpers/number-format-test.js

```diff
@@ -0,0 +1,17 @@
+import { numberFormat } from 'dummy/helpers/number-format';
+import { module, test, skip } from 'qunit';
+
+module('Unit | Helper | number format');
+
+// Works in Chrome but doesn't work in terminal
```

Comment:
Per the issue in the linked ticket: according to the MDN docs the function parameters were added later, and compatibility with that would be around the edge of our supported browser range. 
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#Browser_compatibility

But you don't appear to be using anything more than the stock older variant; I'm a little curious what the failure message looks like. So long as it's purely PhantomJS being kooky, then this is ok to skip for now- I'll make a note to take a look during the next review.